adapters/networkhandler/genericNetworkResponseHandler.js

```diff
@@ -0,0 +1,35 @@
+const { isEmpty } = require("../../v0/util/index");
+/**
+ * network handler as a fall back for all destination nethandlers, this file provides abstraction
+ * for all the network comms btw dest transformer.
+ *
+ * --responseTransform-- this is a function which can be used to handle responses which are not-compatible with
```

Comment:
We probably would need to mention what `not compatible with rudder-server` means. Take an example of a destination like BQStream/Marketo wherein the status codes are not rudder-server friendly(I mean rudder-server by itself cannot conclude if the error is retryable/abortable/throttable). Let's add it a bit later

`Adding the comment here as a note`

Reply:
yep